jax/_src/numpy/reductions.py

```diff
@@ -1249,6 +1249,79 @@ def nanmean(a: ArrayLike, axis: Axis = None, dtype: DTypeLike | None = None, out
 def nanvar(a: ArrayLike, axis: Axis = None, dtype: DTypeLike | None = None, out: None = None,
```

Comment:
Please remove the `@implements` decorator above

Reply:
Done